### PR TITLE
Homewokr 3 is done

### PR DESCRIPTION
**Pull request by Andranik-Arakelyan**

### CODE REVIEW

File: homework-3/scripts/renderTriangle.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+function createTriangle(num) {
```
Comment (ao-alex) [inline homework-3/scripts/renderTriangle.js:1]: better this function be pure function, without side effects, 
meaning its call shouldn't change anything. 
So let it return result, without knowing about anything about dom.

File: homework-3/scripts/renderTriangle.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      inner += " ";
+    }
+    for (let k = num - i - 1; k < num; k++) {
+      inner += " *";
+    }
+    preNode.innerText += inner + "\n";
+  }
+}
+
+function rotateTriangle(num) {
```
Comment (ao-alex) [inline homework-3/scripts/renderTriangle.js:15]: same here

**Comment by ao-alex** [comment]
looks good

File: homework-3/scripts/renderTriangle.js (hunk 2 above)
Comment (Andranik-Arakelyan) [reply homework-3/scripts/renderTriangle.js:15]: Ok, I'll change that part

**Pull request merged**